hack/remote-secret-secret.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Secret
```

Comment:
would it be better to have this in `samples` dir? Can you please also create some `RemoteSecret` sample yaml?

Reply:
good catch